services/frontend-v3/src/components/profileForms/employmentDataForm/EmploymentDataFormView.jsx

```diff
@@ -446,7 +446,7 @@ const EmploymentDataFormView = (props) => {
     return (
       <div style={styles.content}>
         <Title level={2} style={styles.formTitle}>
-          3. <FormattedMessage id="setup.employment" />
```

Comment:
Changing the numbering here will break it for create profile since it is a shared form. Maybe we should remove numbering or pass it in from the layout page?

Reply:
I just tested (nuked the db) and when going through the create profile flow, the numbering was still okk surprisingly